### DISCUSSION TITLE
Line Trigger

### DISCUSSION THREAD

**Discussion by kelepon** [roboflow/supervision#334] [Q&A]
Hello,

How can I ensure that the line is triggered when the start of the bounding box makes contact with the line, rather than having the bounding box completely cross the line?

I'm using a close-range camera to detect trucks.

Thanks :)

**Comment by AlainPilon** [comment]
Hi. I am about to add this feature to my version of the LineCounter. Basically, I think you just need to update this code to trigger your callback: 

https://github.com/AlainPilon/better_line_counter/blob/main/line_counter.py#L56-L57

**Comment by hardikdava** [comment]
@kelepon @AlainPilon We are working on improving `sv.LineZone` with it you will have options to choose different anchor points of the bounding boxes such as upper left corner, lower right corner, center point, etc. You can check the progress #266 . We will soon release this feature. Stay tune!
